samples/nrf9160/fmfu_smp_svr/prj.conf

```diff
@@ -41,3 +41,9 @@ CONFIG_NETWORKING=y
 CONFIG_NET_SOCKETS=y
 CONFIG_NET_SOCKETS_OFFLOAD=y
 CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MCUMGR_CMD_IMG_MGMT=y
+CONFIG_MCUMGR_CMD_OS_MGMT=y
+CONFIG_MCUMGR_CMD_STAT_MGMT=y
+
+# Ensure an MCUboot-compatible binary is generated.
```

Comment:
Should this be here? If so we should include the rationale in the commit message.

Reply:
Copy pasta will remove